src/pages/Index.vue

```diff
@@ -2,18 +2,115 @@
   <Layout>
 
     <!-- Learn how to use images here: https://gridsome.org/docs/images -->
-    <h1>Welcome to the Galaxy Mentoring Network</h1>
+    <g-image alt="Example image" src="~/favicon.png" width="135" />
+    <header class="header">
+        <b-container class=".b-container">
+        <b-row>
+          <b-col lg="6" md="6" sm="12">
+        <h1>Mentorship...for you!</h1>
 
-    <p>Mentorship...for you!</p>
+        <p>An all in one program targeted to speed up your growth!</p>
+        <b-btn class=".btn">Apply Now</b-btn>
+            </b-col>
+            <b-col>
+              <img src="./assets/images/illustration.png" img-alt="two people sitting in front of a computer" width="400px">
```

Comment:
The path to images are wrong. Could you modify them?

Reply:
this should be `./images/illustration.png` right?